File: TP3/JAVA/src/main/resources/Enseignant.java
package src;

public class Enseignant extends Personne {

    private double salaire;

    public Enseignant(String nom, int age, double salaire) {
        super(nom, age);
        this.salaire = salaire;
    }

    public double getSalaire() {
        return salaire;
    }

    @Override
    public void afficherDetails() {
        System.out.println("Enseignant: " + getNom()
                + ", Age: " + getAge()
                + ", Salaire: " + salaire);
    }
}

File: TP3/PYTHON/src/codes_initiaux/enseignant.py
from src.codes_initiaux.personne import Personne

class Enseignant(Personne):
    def __init__(self, nom, age, salaire):
        super().__init__(nom, age)
        self.salaire = salaire

    def afficher_details(self):
        return f"Enseignant: {self.get_nom()}, Age: {self.get_age()}, Salaire: {self.salaire}"